src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule72thesummaryfragment/InputCorrectSummaryJavaDocCheck.java

```diff
@@ -29,63 +30,66 @@ void foo5() {}
      */
     void foo6() {}
 
-    /** 
-     * <a href="mailto:vlad@htmlbook.ru"/> 
+    /**
+     * This is valid.
+     * <a href="mailto:vlad@htmlbook.ru"/>.
```

Comment:
If you fix a 'bad' javadoc in a good 'input' file please make sure to make a copy of it in the bad 'input' file if it isn't already.